lib/block-supports/typography.php

```diff
@@ -15,12 +15,17 @@ function gutenberg_register_typography_support( $block_type ) {
 		return;
 	}
 
-	$has_font_size_support       = _wp_array_get( $block_type->supports, array( 'fontSize' ), false );
-	$has_font_style_support      = _wp_array_get( $block_type->supports, array( '__experimentalFontStyle' ), false );
-	$has_font_weight_support     = _wp_array_get( $block_type->supports, array( '__experimentalFontWeight' ), false );
-	$has_line_height_support     = _wp_array_get( $block_type->supports, array( 'lineHeight' ), false );
-	$has_text_decoration_support = _wp_array_get( $block_type->supports, array( '__experimentalTextDecoration' ), false );
-	$has_text_transform_support  = _wp_array_get( $block_type->supports, array( '__experimentalTextTransform' ), false );
+	$typography_supports = _wp_array_get( $block_type->supports, array( 'typograpgy' ), false );
+	if ( ! $typography_supports ) {
+		return;
+	}
+
+	$has_font_size_support       = _wp_array_get( $typography_supports, array( 'fontSize' ), false );
```

Comment:
It looks like we miss here the check for the font family? It's previous to this PR, so fine to add after as well.